haystack_experimental/dataclasses/chat_message.py

```diff
@@ -67,6 +69,98 @@ def __repr__(self) -> str:
         fields_str = ", ".join(fields)
         return f"{self.__class__.__name__}({fields_str})"
 
+    @classmethod
+    def from_file_path(
```

Comment:
Did we want to also support PDF files with this utility method?

Reply:
To keep these convenience methods simple, I would prefer to point users to the appropriate components. WDYT?

Reply:
It's probably fine for now, but I'd make it clear in the docs which file types these convenience methods support. Then we can see how often they get used and if there is a demand for one that supports PDFs as well.

Reply:
I tried to improve the docstrings about this aspect